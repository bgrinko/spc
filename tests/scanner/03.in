var
    a, b, c: integer;
begin
    e = 12;
    c = 1.2;
end.