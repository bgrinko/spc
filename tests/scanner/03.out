1
1	3	NONE		var
3	5	NONE		a
3	6	NONE		,
3	8	NONE		b
3	9	NONE		,
3	11	NONE		c
3	13	NONE		:
3	21	NONE		integer
4	-1	NONE		;
5	5	NONE		begin
7	5	NONE		e
7	8	NONE		=
7	12	NONE		12
8	-1	NONE		;
9	5	NONE		c
9	8	NONE		=
9	14	NONE		1.2
10	-1	NONE		;
11	3	NONE		end
11	4	NONE		.
